recipes/cccl/all/conandata.yml

```diff
@@ -2,3 +2,9 @@ sources:
   "1.1":
     url: https://github.com/swig/cccl/archive/cccl-1.1.tar.gz
     sha256: 03ed67d04e8b1e165b8f8d42546ab62ff6c42b3623a2358a7c8255ced144ce28
+  "1.2":
```

Comment:
I did not check so I sorry if this is a repeate, we usually do not add inbetween releases only the latest

Is there a specific reason you need this one?

Reply:
I thought it was the latest before I checked their repo and noticed 1.3 is actually out, just without a release tag 😅 I can remove it if you prefer